.github/workflows/publish-command.yml

```diff
@@ -109,7 +109,8 @@ jobs:
           ZENDESK_TALK_TEST_CREDS: ${{ secrets.ZENDESK_TALK_TEST_CREDS }}
           ZOOM_INTEGRATION_TEST_CREDS: ${{ secrets.ZOOM_INTEGRATION_TEST_CREDS }}
           PLAID_INTEGRATION_TEST_CREDS: ${{ secrets.PLAID_INTEGRATION_TEST_CREDS }}
-          DESTINATION_S3_INTEGRATION_TEST_CREDS: ${{ secrets.DESTINATION_S3_INTEGRATION_TEST_CREDS }}
+          DESTINATION_S3_CSV_INTEGRATION_TEST_CREDS: ${{ secrets.DESTINATION_S3_CSV_INTEGRATION_TEST_CREDS }}
```

Comment:
The format config is different:

One is:

```json
"format": {
  "format_type": "CSV",
  "flattening": "Root level flattening"
}
```

The other is:

```json
"format": {
  "format_type": "Parquet",
  "compression_codec": "GZIP"
}
```

See the two configs in `sample_secrets` for details.

This is similar to why we need two secrets for Redshift.

Reply:
I think we can have only 1 set of credentials and change the cofig method to only read the relevant information from the credentials. 
For instance this part can be hardcoded in the test class in the config i.e. for CSV test it can be 
```
"format": {
    "format_type": "CSV",
    "flattening": "Root level flattening"
  }
```
and for Parquet test it can be 
```
"format": { "format_type": "Parquet", "compression_codec": "GZIP" }
```

and the sensitive information can be populated from the credentials.

Reply:
Good idea. Done.